### ISSUE TITLE
Count of Pass, N/A and Fail when entering checkpoints

### ISSUE THREAD

**Issue by irwink** [wet-boew/web-reporting#52]
It would be helpful to have a summary table on the form when entering WCAG results to list the number of checkpoints marked as Pass, N/A and Fail.  This would be used when reviewing results to ensure a checkpoint is not missed or if one is not recorded correctly (e.g. a Pass recorded as N/A).

Page:
https://publiservice.tbs-sct.gc.ca/ws-nw/wr-rw/administration/accessibility-accessibilite/checklist-grille.aspx?lang=eng

**Comment by LaurentGoderre** [comment]
@irwink you are entirely right
